records_mover/records/mover.py

```diff
@@ -2,7 +2,7 @@
                       SupportsToFileobjsSource,
                       FileobjsSource, SupportsToDataframesSource)
 from .targets.base import (RecordsTarget, SupportsMoveFromRecordsDirectory,
-                           SupportsMoveFromTempLocAfterFillingIt,
+                           MightSupportMoveFromTempLocAfterFillingIt,
```

Comment:
This is another one of those interfaces implemented by sources and targets (in this case, targets).  I renamed this interface to be more tentative - as now it can tell you at runtime whether it is able to do do what it says on the tin.

Specifically, if it can do it, this interface has a function which tells the target to:
1. create a temporary location that the target can load data from efficiently (e.g., an S3 bucket directory that Redshift can run a COPY statement from)
2. tell the source to fill in that temporary location with a records directory (e.g., copy a CSV file from your local disk to that temporary S3 directory and add a manifest that Redshift likes)
3. load from that temporary location (e.g., run the Redshift COPY command)